File: app/src/androidTest/java/me/kirimin/mitsumine/test/suite/UnitTestSuite.java
package me.kirimin.mitsumine.test.suite;

import junit.framework.Test;
import junit.framework.TestCase;
import junit.framework.TestSuite;

import me.kirimin.mitsumine.test.FeedListFilterTest;

public class UnitTestSuite extends TestCase {

    public static Test suite() {
        TestSuite suite = new TestSuite();
        suite.addTestSuite(FeedListFilterTest.class);
        return suite;
    }
}

File: app/src/androidTest/java/me/kirimin/mitsumine/test/suite/UnitTestSuite.kt
package me.kirimin.mitsumine.test.suite

import org.junit.runner.RunWith
import org.junit.runners.Suite

import me.kirimin.mitsumine.test.BookmarkTest
import me.kirimin.mitsumine.test.FeedUtilTest

RunWith(Suite::class)
Suite.SuiteClasses(FeedUtilTest::class, BookmarkTest::class)
public class UnitTestSuite